inference-engine/include/ie_blob.h

```diff
@@ -756,6 +826,30 @@ class TBlob : public MemoryBlob {
     void* getHandle() const noexcept override {
         return _handle;
     }
+
+    /**
+     * @brief Creates an new empty rvalue LockedMemory object.
+     *
+     * @return rvalue for the empty locked object of type T
+     */
+    virtual LockedMemory<T> data() const noexcept {
+        return std::move(lockme<T>());
+    }
+
+    /**
+     * @brief Clones Blob object of the particular class. Clone references the same memory as this object.
+     *
+     * @return Pointer to the cloned object
+     */
+    Blob* clone() const override {
+        TBlob* cloned = new TBlob<T>(this->tensorDesc, this->data(), this->byteSize());
```

Comment:
I suppose, `this->size()` should be used instead of `this->byteSize()` according to the constructor documentation.

```
     * @param data_size Length of the pre-allocated array. If not set, size is assumed equal
     * to the dot product of dims.
```